contentcuration/contentcuration/serializers.py

```diff
@@ -457,7 +457,7 @@ def update(self, instance, validated_data):
         return instance
 
     def get_node_ancestors(self, node):
-        return node.get_ancestors().values_list('id', flat=True)
+        return list(node.get_ancestors().values_list('id', flat=True))
```

Comment:
What's the reason we needed to turn this into a list? Seems more efficient to keep it as an iterator so it's not all loaded in memory

Reply:
This is because the stock JSON serializer doesn't properly serialize iterators into a list. This is done when we pass the task results into the task object. We won't know which DRF serializer to use, so we serialize the object to straight JSON instead. The purpose of all this is so that we can properly update the UI without refreshing the page.